packages/frontend/src/hooks/useQueryResults.tsx

```diff
@@ -82,6 +83,7 @@ const getQueryResults = async ({
     const timezoneFixQuery = query && {
```

Comment:
There are some other places where we do something similar to this. I'm just noting that we should update those too (ex useCompliedSQL, useCalculateTotals)